features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml

```diff
@@ -291,70 +345,81 @@
-            <Scope displayName="Update OIDC Scopes" name="internal_oidc_scope_mgt_update"/>
-            <Scope displayName="Delete OIDC Scopes" name="internal_oidc_scope_mgt_delete"/>
+            <Scope displayName="View OIDC Scopes" name="internal_oidc_scope_mgt_view" 
+                    description="Create new OAuth2 scopes and bindings"/>
+            <Scope displayName="Create OIDC Scopes" name="internal_oidc_scope_mgt_create" 
+                    description="View OAuth2 scopes and bindings"/>
+            <Scope displayName="Update OIDC Scopes" name="internal_oidc_scope_mgt_update" 
+                    description="Update OAuth2 scopes and bindings"/>
+            <Scope displayName="Delete OIDC Scopes" name="internal_oidc_scope_mgt_delete" 
+                    description="Delete OAuth2 scopes and bindings"/>
         </Scopes>
     </APIResource>
     <APIResource name="Tenant Management API" identifier="/api/server/v1/tenants"
                  requiresAuthorization="true" description="API representation of the Tenant Management API" type="SYSTEM">
         <Scopes>
-            <Scope displayName="List Tenants" name="internal_list_tenants"/>
-            <Scope displayName="Modify Tenants" name="internal_modify_tenants"/>
+            <Scope displayName="List Tenants" name="internal_list_tenants" 
+                    description="Create/retrieve the organizations (root) and check domain availability"/>
+            <Scope displayName="Modify Tenants" name="internal_modify_tenants" 
+                    description="Activate/deactivate the organization (root) and delete its meta data"/>
         </Scopes>
     </APIResource>
     <APIResource name="Organization Discovery API"
                  identifier="/api/server/v1/organizations/discovery"
                  requiresAuthorization="true"
                  description="API representation of the Organization Discovery API" type="TENANT_ADMIN">
         <Scopes>
-            <Scope displayName="View Organization Discovery"
-                   name="internal_organization_discovery_view"/>
-            <Scope displayName="Update Organization Discovery"
-                   name="internal_organization_discovery_update"/>
-            <Scope displayName="Delete Organization Discovery"
-                   name="internal_organization_discovery_delete"/>
+            <Scope displayName="View Organization Discovery" name="internal_organization_discovery_view" 
+                    description="View discovery attributes of the organization (root)"/>
+            <Scope displayName="Update Organization Discovery" name="internal_organization_discovery_update" 
+                    description="Update discovery attributes of the organization (root)"/>
+            <Scope displayName="Delete Organization Discovery" name="internal_organization_discovery_delete" 
+                    description="Delete discovery attributes of the organization (root)"/>
```

Comment:
this API can view, update, delete discovery attributes of any organization within the root organization. Therefore the description should be updated to convey that meaning

Reply:
Fixed